src/main/java/com/amadeus/booking/FlightOrder.java

```diff
@@ -0,0 +1,64 @@
+package com.amadeus.booking;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.Response;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.Resource;
+
+/**
+ * <p>
+ *   A namespaced client for the
+ *   <code>/v1/booking/flight-orders:flightOrderId</code> endpoints.
+ * </p>
+ *
+ * <p>
+ *   Access via the Amadeus client object.
+ * </p>
+ *
+ * <pre>
+ * Amadeus amadeus = Amadeus.builder("clientId", "secret").build();
+ * amadeus.booking.flightOrder;</pre>
+ */
+public class FlightOrder {
+  private Amadeus client;
+  private String flightOfferId;
+
+  /**
+   * Constructor.
+   * @hide
+   */
+  public FlightOrder(Amadeus client, String flightOfferId) {
+    this.client = client;
+    this.flightOfferId = flightOfferId;
+  }
+
+  /**
+   * <p>
+   *   Allows you to manipulate a flight order.
+   * </p>
+   *
+   * <pre>
+   *  com.amadeus.resources.FlightOrder order = amadeus.booking.flightOrder.(
+   * "eJzTd9f3NjIJdzUGAAp%2fAiY=").get();
+   * </pre>
+   * @param params the parameters to send to the API
+   * @return an API response object
+   * @throws ResponseException when an exception occurs
+   */
+
+  public com.amadeus.resources.FlightOrder get(Params params) throws ResponseException {
+    String path = String.format("/v1/booking/flight-orders/%s", flightOfferId);
+    Response response = client.get(path, params);
+    return (com.amadeus.resources.FlightOrder) Resource.fromObject(
+        response, com.amadeus.resources.FlightOrder.class);
+  }
+  
+  /**
+   * Convenience method for calling <code>get</code> without any parameters.
+   * @see com.amadeus.booking.FlightOrder#get()
```

Comment:
Do you mean "@see com.amadeus.resources.FlightOrder get(Params params)"?

Reply:
This is the method for calling get without any params. Check [here](https://github.com/amadeus4dev/amadeus-java/blob/master/src/main/java/com/amadeus/shopping/HotelOffer.java#L59) for example.